### PR TITLE
feat(DASH): update period as part of configuration

### PR DESCRIPTION
**Pull request by Iragne**
Closes https://github.com/shaka-project/shaka-player/issues/6418

### CODE REVIEW

File: externs/shaka/player.js [hunk 1]
```diff
@@ -970,6 +971,8 @@ shaka.extern.xml.Node;
  *   but may raise issues if manifest does not have stream consistency
  *   between periods.
  *   Defaults to <code>false</code>.
+ * @property {number} updatePeriod
+ *   Override the update time for the manifest
```
Comment (avelad) [inline externs/shaka/player.js:975]: Please add a better description and put the default value.

File: lib/util/player_configuration.js [hunk 2]
```diff
@@ -112,6 +112,7 @@ shaka.util.PlayerConfiguration = class {
       segmentRelativeVttTiming: false,
       raiseFatalErrorOnManifestUpdateRequestFailure: false,
       dash: {
+        updatePeriod: -1,
```
Comment (avelad) [inline lib/util/player_configuration.js:115]: Add it to the end of all dash configs

**Review by avelad — CHANGES_REQUESTED** [review]
Add the new config in the Demo.

File: lib/dash/dash_parser.js [hunk 3]
```diff
@@ -1689,6 +1694,16 @@ shaka.dash.DashParser = class {
     this.setUpdateTimer_(updateDelay);
   }
 
+  /**
+   * Update now the manifest
+   *
+   * @private
+   */
+  updateNow_() {
+    console.log('updateNow');
```
Comment (avelad) [inline lib/dash/dash_parser.js:1703]: Remove it

Comment (Iragne) [reply lib/dash/dash_parser.js:1703]: I'm really sorry, i push it too fast and just realized all of it too late

File: lib/util/player_configuration.js (hunk 2 above)
Comment (Iragne) [reply lib/util/player_configuration.js]: just added

File: externs/shaka/player.js (hunk 1 above)
Comment (Iragne) [reply externs/shaka/player.js]: Description updated with proper manifest naming and default value

**Comment by Iragne** [comment]
@avelad  just added the config demo

**Review by avelad — APPROVED** [review]

**Pull request merged**